### PR TITLE
chore: removes OAuth2 capabilities from the data sink

### PR DESCRIPTION
**Pull request by wolf4ood**
## What this PR changes/adds

removes OAuth2 capabilities from the data sink

### CODE REVIEW

File: extensions/data-plane/data-plane-http-oauth2-core/src/test/java/org/eclipse/edc/connector/dataplane/http/oauth2/DataPlaneHttpOauth2ExtensionTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2023 Amadeus
```
Comment (paullatzelsperger) [inline extensions/data-plane/data-plane-http-oauth2-core/src/test/java/org/eclipse/edc/connector/dataplane/http/oauth2/DataPlaneHttpOauth2ExtensionTest.java:2]: seems incorrect

**Review by paullatzelsperger — APPROVED** [review]

**Review by jimmarino — APPROVED** [review]

**Comment by florianrusch-zf** [comment]
It would be great to know what the reason for the removal is. I have no argument against the removal, I'm just curious and the PR description is more or less non-existent...

**Comment by jimmarino** [comment]
The reasons for this removal will be explained by the committers later.

**Comment by florianrusch-zf** [comment]
@jimmarino where can I find the explanation?

**Comment by jimmarino** [comment]
This was a security vulnerability that should be described in the release notes. You can look for it there.

**Pull request merged**